app/assets/javascripts/models/user.js

```diff
@@ -1,3 +1,15 @@
 module.exports = Backbone.Model.extend({
-  name: 'user'
+  name: 'user',
+  initialize: function() {
```

Comment:
I was thinking the same, but it keeps giving me the same error if I remove the `initialize`.